### PR TITLE
feat(typekit): added code class

### PR DESCRIPTION
**Pull request by nathanlb**
### Proposed Changes

- Added typekit styling class `.code` which displays text in code format following the same style as our public documentation. 

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/vapor/scss/typekit.scss [hunk 1]
```diff
@@ -96,6 +96,16 @@
     letter-spacing: 0.003em;
 }
 
+%code {
+    padding: 0.25em;
+    color: #626971;
+    font-family: PTMono-Regular, monospace;
+    overflow-wrap: break-word;
+    background-color: #f6f7f9;
+    border: 1px solid #e5e8e8;
```
Comment (toofff) [inline packages/vapor/scss/typekit.scss:105]: Use variable for color please

File: packages/vapor/scss/typekit.scss [hunk 2]
```diff
@@ -96,6 +96,16 @@
     letter-spacing: 0.003em;
 }
 
+%code {
+    padding: 0.25em;
+    color: #626971;
```
Comment (toofff) [inline packages/vapor/scss/typekit.scss:101]: Use variable for color please

File: packages/vapor/scss/typekit.scss [hunk 3]
```diff
@@ -96,6 +96,16 @@
...
+    font-family: PTMono-Regular, monospace;
```
Comment (toofff) [inline packages/vapor/scss/typekit.scss:102]: Why not use this font here `$code-font-family`

**Review by FelixBlaisThon — APPROVED** [review]

File: packages/vapor/scss/typekit.scss [hunk 4]
```diff
@@ -385,3 +395,10 @@ p,
     @extend %book;
     @extend %subdued;
 }
+
+code,
```
Comment (GermainBergeron) [inline packages/vapor/scss/typekit.scss:399]: I wouldn't modify the `code` element, otherwise it's harder to use the styleguide in other websites

Comment (nathanlb) [reply packages/vapor/scss/typekit.scss]: done deal

**Review by toofff — APPROVED** [review]

**Review by GermainBergeron — APPROVED** [review]

**Pull request merged**